src/components/Global/ConfirmInviteModal/index.tsx

```
@@ -1,7 +1,7 @@
 'use client'
 import { type FC } from 'react'
 import Image from 'next/image'
-import { PEANUT_LOGO_BLACK, PEANUTMAN_LOGO } from '@/assets'
+import { PEANUT_LOGO_BLACK, PEANUTMAN } from '@/assets'
```

ESLint

[error] '`@/assets`' import is restricted from being used. Import from a specific file instead of the '`@/assets`' barrel — barrels force the bundler to load every re-export and hurt build perf. See CLAUDE.md.

(no-restricted-imports)